osschallenge/static/osschallenge/stylesheet.css

```diff
@@ -159,6 +159,7 @@ div.detail form input {
 .btn.btn-primary {
     width: 200px;
     margin: 10px 0 0 0;
+    height: 0;
```

Comment:
With `height: 0` the button is probably not visible. Is this intended?

Reply:
The Button is still visible, without ```height: 0``` the button would be a few pixels too big and it wouldn't lineup in a few places.